src/autocomplete/parsers/postgresql/grammar/select/postgresql_select.test.ts

```diff
@@ -251,7 +244,7 @@ test('should suggest keywords, groupBys, orderBys after null safe WHERE', () =>
     };
     expect(parseResult.suggestOrderBys).toEqual(orderBysSuggestion);
 
-    const groupBysSuggestion: GroupBysSuggestion = {
+    const groupBysSuggestion: CommonPopularSuggestion = {
```

Comment:
@NikitaShkaruba didn't like this naming of Common..
to be honest I don't get the word popular here
maybe we should leave aliases as they were?

Reply:
I got you, fixed

Reply:
True, let's leave it as it was